lottie/src/main/java/com/airbnb/lottie/network/CacheProvider.java

```diff
@@ -0,0 +1,9 @@
+package com.airbnb.lottie.network;
+
+import java.io.File;
+
+import androidx.annotation.NonNull;
+
+public interface CacheProvider {
```

Comment:
Javadoc. Also indicate when `getCacheDir()` is called and whether it is called every time or cached.

Reply:
done

Reply:
Let's name this `LottieNetworkCacheProvider`
and in the docs specify that this is the cache where animations downloaded via url are saved.

Reply:
done